perovskite_solar_cell_database/schema_sections/ions/ion.py

```diff
@@ -0,0 +1,166 @@
+from nomad.datamodel.metainfo.basesections import PureSubstanceSection
+from nomad.metainfo import Quantity
+import openpyxl
```

Comment:
I guess this is already one of the dependencies of `nomad-lab`? I would add it in the list of plugin dependencies just in case.